README.md

```diff
@@ -1,81 +1,27 @@
-# 1. Overview
+A crypto trading app demo to show a real-time trading dashboard for three different exchanges.
 
-Demo to show a real-time trading dashboard for three different exchanges.
+![crypto-app-ui](https://user-images.githubusercontent.com/1088136/198411981-2afbdd21-d145-4ad6-aea1-ee933ed482a5.png)
 
 The complete crypto-trading demo has two components:
 
-1. A stream application.
-2. A UI application written in Reactjs
+1. A backend built with Macrometa using a Document Collection, multiple Streams, and a Stream Worker.
 
+If you haven't already configure the backend, refer to the [getting started guide](https://macrometa.com/docs/apps/crypto-trading).
```

Comment:
Rewrite this sentence and put it with the prerequisites below. Also, make prerequisites an h2.

## Prerequisites

- Thing 1
- Thing 2